tardis/apps/publication_workflow/components/utils/PublicationToast.jsx

```diff
@@ -0,0 +1,35 @@
+import { Toast } from 'react-bootstrap';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
import { Toast } from "react-bootstrap";
```